src/components/create_cluster/index.js

```diff
@@ -197,6 +200,31 @@ class CreateCluster extends React.Component {
     });
   };
 
+  getAvailableRegions() {
+      var regions = [
+          'us-east-1',
+          'us-east-2',
+          'us-west-1',
+          'us-west-2',
+          'ap-south-1',
+          'ap-northeast-1',
+          'ap-northeast-2',
+          'ap-northeast-3',
+          'ap-southeast-1',
+          'ap-southeast-2',
+          'ca-central-1',
+          'eu-central-1',
+          'eu-west-1',
+          'eu-west-2',
+          'eu-west-3',
+          'eu-north-1',
+          'sa-east-1',
+          'us-gov-east-1',
+          'us-gov-west-1',
+      ];
+      return regions;
+  }
+
```

Comment:
Will this move to the info endpoint?

Reply:
Yes, I did couple shortcuts here and there in order to get this running E2E within two days :slightly_smiling_face: